index.js

```diff
@@ -28,13 +28,16 @@ class AssetRewrite extends Filter {
     this.assetMap = options.assetMap || {};
     this.prepend = options.prepend || '';
     this.ignore = options.ignore || []; // files to ignore
+    this.enableCaching = options.enableCaching || false;
 
     this.assetMapKeys = null;
   }
 
 
   processAndCacheFile(srcDir, destDir, relativePath) {
-    this._cache = new Cache();
+    if (!this.enableCaching) {
```

Comment:
I find this vaguely confusing to read (though I know it is correct). Maybe make a local variable to help give a name that helps?

```js
let shouldResetCache = !this.enableCaching;
if (shouldResetCache) {
  this._cache = new Cache();
}
```

What do you think?

Reply:
👍  Updated